CIP-0010/registry.schema.json

```diff
@@ -1,64 +1,25 @@
-    "$id": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0010",
-    "type": "array",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0010",    
     "title": "Transaction Metadata Label Registry",
-    "description": "JSON schema for transaction metadata label registry",
-    "default": [],
-    "examples": [
-        [
-            {
-                "transaction_metadatum_label": 1967,
-                "description": "nut.link metadata oracles registry"
-            },
-            {
-                "transaction_metadatum_label": 1968,
-                "description": "nut.link metadata oracles data points"
-            }
-        ]
-    ],
-    "additionalItems": false,
+    "description": "JSON schema for transaction metadata label registry",    
+    "type": "array",
     "items": {
-        "$id": "#/items",
-        "anyOf": [
-            {
-                "$id": "#/items/anyOf/0",
-                "type": "object",
-                "title": "The first anyOf schema",
-                "description": "An entry in the transaction metadata label registry",
-                "default": {},
-                "examples": [
-                    {
-                        "transaction_metadatum_label": 1967,
-                        "description": "nut.link metadata oracles registry"
-                    }
-                ],
-                "required": [
-                    "transaction_metadatum_label",
-                    "description"
-                ],
-                "properties": {
-                    "transaction_metadatum_label": {
-                        "$id": "#/items/anyOf/0/properties/transaction_metadatum_label",
-                        "type": "integer",
-                        "title": "The transaction_metadatum_label number",
-                        "default": 0,
-                        "examples": [
-                            1967
-                        ]
-                    },
-                    "description": {
-                        "$id": "#/items/anyOf/0/properties/description",
-                        "type": "string",
-                        "title": "The metadatum label description",
-                        "default": "",
-                        "examples": [
-                            "nut.link metadata oracles registry"
-                        ]
-                    }
-                },
-                "additionalProperties": true
-            }
-        ]
+      "type": "object",
+      "properties": {
+        "transaction_metadatum_label": {
+          "type": "integer",
+          "minimum": 0
+        },
+        "description": {
+          "type": "string",
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "transaction_metadatum_label",
+        "description"
+      ],
+      "additionalProperties": false
```

Comment:
Is there rationale for changing this?

Reply:
This actually makes sense in my opinion. We do NOT want potentially malicious or harmful content to be added to this registry file (like people advertising their URLs, etc) without at least needing to pass through the lens of the CIP Editor body and potentially an update being issued to the schema itself.

I'm in favor of leaving it so that any and all deviations from the schema get flagged and we can decide on them on a case-by-case basis.

Reply:
Yup agreed!
appreciate the explanation